hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -219,13 +230,13 @@ public static JavaSparkContext buildSparkContext(String appName, String sparkMas
   /**
    * Build Hoodie write client.
    *
-   * @param jsc Java Spark Context
-   * @param basePath Base Path
-   * @param schemaStr Schema
+   * @param jsc         Java Spark Context
+   * @param basePath    Base Path
+   * @param schemaStr   Schema
    * @param parallelism Parallelism
    */
   public static HoodieWriteClient createHoodieClient(JavaSparkContext jsc, String basePath, String schemaStr,
-      int parallelism, Option<String> compactionStrategyClass, TypedProperties properties) {
+                                                     int parallelism, Option<String> compactionStrategyClass, TypedProperties properties) {
```

Comment:
I think there is a setting not to format the lines untouched.. may be we can avoid all these whitespace changes that way..